systemtest/src/test/java/io/strimzi/systemtest/SecurityST.java

```diff
@@ -877,6 +879,93 @@ void testTlsHostnameVerificationWithMirrorMaker() {
         testMethodResources().deleteMirrorMakerWithoutWait(kafkaMirrorMaker);
     }
 
+    @Test
+    void testAclRuleWrite() throws Exception {
```

Comment:
The test seems to test both read and write. So the name is a bit misleading.

Reply:
yeah...Firstly i only test the write `ACL`, then i create new test case for read and then i realized that i should use only one test case, which will take care of these two properties instead of deploying two times kafka(takes much time).